Public Relations Approval Process.md

```diff
@@ -0,0 +1,8 @@
+**Publication Approval Process**
+
+1.  Team member drafts content for release (this includes posters, presentations, manuscripts, and anything that will be released publicly that is branded KBase).
```

Comment:
Hi!  Perhaps it is a poor word choice: currently, we intend for all posters and presentations referencing KBase, publications facilitated by and citing KBase, handouts about KBase, etc., to come through PR-Approval.  Not only will the members of PR-Approval be vetting the docs for sensitive/inappropriate content, but they will also do some editing before any untoward typos reach the public.